tests/test_matrix_utils_room.py

```
@@ -648,3 +651,188 @@ def test_display_room_channel_mappings_no_display_name():
+        "mmrelay.matrix_utils.matrix_client", mock_client, raising=False
+    )
+
+    result = await get_displayname("@user:example.org")
+
+    assert result == "Alice"
+
+
+@pytest.mark.asyncio
+@patch("mmrelay.matrix_utils.matrix_client")
+@patch("mmrelay.matrix_utils.logger")
+async def test_get_user_display_name_room_name(_mock_logger, _mock_matrix_client):
```

Ruff (0.15.9)

[warning] Fixture `_mock_logger` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)